dbt/models/ccao/docs.md

```diff
@@ -1,6 +1,3 @@
-**Primary Key**: `reascd`
-{% enddocs %}
-
```

Comment:
Nothing to do with the PR, just random extraneous code that needed to be removed.